Dockerfile

```diff
@@ -14,4 +14,11 @@ RUN apt-get update && \
     rm -fr /tmp/sqliteodbc-0.9996 && \
     apt-get purge -y build-essential sqlite3 libsqlite3-dev unixodbc-dev && \
     rm -rf /var/cache/apk/*
-    
+ 
+# Install Python, pip and requests library 
+RUN apt-get update && \
+    apt-get install -y python3-pip python3-dev  && \
+    cd /usr/local/bin && \
+    ln -s /usr/bin/python3 python && \
```

Comment:
I know this is only proxy, but are you sure you want to make Python 3 as a default interpreter? Are there any other scripts that use Python 2? The default interpreter can also be defined in the script's shebang line.

Reply:
To my knowledge there are currently no python scripts in proxy and this is the reason Python has to be installed there in the first place. I figured Python3 would be better choice as 2 is no longer supported. I can't think of a reason to choose Python2 instead of Python3.

Reply:
Agreed.